app/helpers/ops_helper/role_rbac_details_helper.rb

```diff
@@ -22,6 +22,16 @@ def rbac_role_info_view(role, rbac_menu_tree)
                         })
   end
 
+  def select_tree_node(tree_id)
```

Comment:
Very minor but you missed the `_onclick` suffix on this one...since you're going to do a follow up it can wait for the follow up.